src/store/session.store.ts

```diff
@@ -3,28 +3,28 @@ import { Collection } from 'mongodb'
 
 import { mongoService } from '../services/mongo.service'
 
-interface SessionCollection {
+interface SessionItem {
```

Comment:
Can you revert these 2 store files? I've got a bigger change in the pipe that's going to conflict.